module/VuFindSearch/src/VuFindSearch/Backend/EDS/Base.php

```diff
@@ -49,7 +49,9 @@ abstract class Base
     protected $debug = false;
 
     /**
-     * EDSAPI host
+     * EDSAPI host.  Although the default is the same as $sessionHost,
```

Comment:
This comment says the two values are identical, but the values below are not identical. What is intended/expected?

Reply:
Sorry, I think the defaults were identical, but my comment made everything even more confusing, as did the $edsApiHost variable which was now used for both EDS and EPF endpoints.  I renamed it to $apiHost and removed the default value from Backend altogether, since there is no longer a reasonable default.  I realize that's a BC break if they don't have it set in the .ini files (which would be odd since it's there by default, but fine).  So I added the defaults to the two BackendFactory classes instead.

Reply:
I can add a changelog note about this as well, just to be on the safe side.